implementations/rust/ockam/ockam_command/src/project/addon/configure_kafka/aiven.rs

```diff
@@ -0,0 +1,25 @@
+use clap::Args;
+
+use crate::project::addon::configure_kafka::{run_impl, KafkaCommandConfig};
+use crate::util::node_rpc;
+use crate::{docs, CommandGlobalOpts};
+
+const LONG_ABOUT: &str = include_str!("../static/configure_aiven/long_about.txt");
+const AFTER_LONG_HELP: &str = include_str!("../static/configure_aiven/after_long_help.txt");
+
+/// Configure the Aiven (Kafka) addon for a project
```

Comment:
@adrianbenavides Just curious why some of the help strings say "<product> (Kafka)", and others just say "<product>".

Eg of "<product> (Kafka)": 
- `/// Configure the Aiven (Kafka) addon for a project`
- `/// Configure the Instaclustr (Kafka) addon for a project`

Eg of "<product>":
- `/// Configure the Confluent addon for a project`
- `/// Configure the Redpanda addon for a project`
- `/// Configure the WarpStream addon for a project`

Reply:
Not sure about that, that was written but @glenngillen . I thought it was on purpose, but if Glenn can confirm would be great.